readthedocs/telemetry/collectors.py

```diff
@@ -92,8 +92,27 @@ def collect(self):
             },
         }
         data["doctool"] = self._get_doctool()
+
+        # NOTE: Check if there are files at `_build/html` (the old output directory)
+        # and log these projects so we can communicate with their maintainers
+        #
+        # This temporal and should be removed in the future once we have
+        # decided what to do with this collected data.
```

Comment:
We probably aren't as concerned about collecting this data as we are about saving user builds from having bad content. Anyone using doxygen and friends could have `_build/html` hardcoded, and will have silently broken docs.